pull_request_template.md

```diff
@@ -1,11 +1,37 @@
 ## Summary
 
-[Change!] Describe your feature, problems you had, notes, improvements and others.
+![](https://github.trello.services/images/mini-trello-icon.png) [Get random qoute from quotes garden](https://trello.com/c/vip1nkUs/32-servicios-promises-witter)
```

Comment:
This template have to remain the same for every Pull Request and then have to be modified in the PR in GitHub.
Also, you do not need test plan or screenshots for backend, but you do need the link to the developed card (in order to make the reviewer able to read it)